.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Environment
+* SuperCollider version:
+* Operating system and version:
+<!--- Include any other relevant details about your environment (Qt version, audio driver, etc.) -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Steps to reproduce (for bugs)
+
+```supercollider
+// Please paste SuperCollider code here.
+// Try to make your example as minimal as possible.
+```
+
+## Error message (for bugs)
+
+```
+// Please paste any error messages here.
```

Comment:
add "including the full stack trace"

Beginners usually just literally post the error message.